tail.go

```diff
@@ -57,12 +58,13 @@ type logger interface {
 // Config is used to specify how a file must be tailed.
 type Config struct {
 	// File-specifc
-	Location    *SeekInfo // Seek to this location before tailing
-	ReOpen      bool      // Reopen recreated files (tail -F)
-	MustExist   bool      // Fail early if the file does not exist
-	Poll        bool      // Poll for file changes instead of using inotify
-	Pipe        bool      // Is a named pipe (mkfifo)
-	RateLimiter *ratelimiter.LeakyBucket
+	Location     *SeekInfo // Seek to this location before tailing
+	LineLocation *SeekInfo // Seek to this line number before tailing
```

Comment:
@nishantroy To make this work just set this LineLocation in the config. It works the the same way Location works, but instead of moving by position it moves by \n's. 

Also see the README on the underlying library https://github.com/StoicPerlman/fls. If you have trouble using it just shoot me a message.